resqpy/time_series/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Time series module"""
+
+__all__ = [
+    'time_series', 'time_series_from_nexus_summary', 'time_duration', 'any_time_series_base_resqpy', 'TimeSeries'
```

Comment:
This list should contain the previously visible classes and functions without any name changes (unless we need to make a breaking change for a good reason).  By my reckoning: ['AnyTimeSeries', 'GeologicTimeSeries', 'TimeDuration', 'TimeSeries', 'selected_time_series', 'simplified_timestamp', 'cleaned_timestamp', 'time_series_from_list', 'merge_timeseries_from_uuid', 'time_series_from_nexus_summary', 'geologic_time_str', 'timeframe_for_time_series_uuid', 'any_time_series']

And then each of those must be imported from its new home in the from... statements below.